urdf2webots/parserURDF.py

```diff
@@ -396,15 +397,18 @@ def export(self, file, indentationLevel):
         if self.numberOfLayers:
             file.write(indentationLevel * indent + '  numberOfLayers %d\n' % self.numberOfLayers)
         if self.minRange:
-            if self.minRange < 0.01:
-                file.write(indentationLevel * indent + '  near %lf\n' % self.minRange)
```

Comment:
`minRange` and `near` default values are `0.01` in Webots, with the constraint that `near <= minRange`.

Thus we should set:
-  `near = minRange` if `minRange` < 0.01 and `near` is not defined.
-  `minRange = near` if `near` > 0.01 and `minRange` is not defined.

Reply:
Thanks for your feedback! Implemented in https://github.com/cyberbotics/urdf2webots/pull/160/commits/136a429a934a4d1cb6f6db4647f689ae3fd60582